lib/src/metrics/weight-of-class/weight_of_class_metric.dart

```diff
@@ -0,0 +1,38 @@
+import 'package:analyzer/dart/ast/ast.dart';
+import 'package:code_checker/checker.dart';
+
+import '../../utils/declaration_utils.dart';
+
+class WeightOfClassMetric {
+  double compute(
+    ScopedClassDeclaration component,
```

Comment:
Why classes are called components?

Reply:
fix

Reply:
how should we handle other 'component' references in the codebase? different request?

Reply:
exactly

Reply:
still component reference https://github.com/wrike/dart-code-metrics/pull/196/files#diff-640bb8695d3503dd75b8fff7609d52ce2ff936ac4b9fffe67342158fee786059R35

Reply:
could u please create an issue for renaming then?